src/service/config.proto

```diff
@@ -40,27 +61,33 @@ message protocol_http_roaming_v1 {
   string path = 3;           // path component of URL of roaming partner
 }
 
+// Server Route definition
 message server_v1 {
-  bytes host = 1;
-  uint32 port = 2;
+  bytes host = 1;  // ex: endpoint.test.com
```

Comment:
I'm trying to remember if there was an argument for having these fields as `bytes` rather than `string`.
It would really help clear up the confusion between fields like `server_v1.host` or `route_v1.id` which are strings, and `owner`, `payer`, `signer` that are pubkey bytes that cannot be stringified.

Reply:
agreed